invokeai/frontend/web/src/features/nodes/util/graph/generation/buildFLUXGraph.ts

```diff
@@ -125,9 +138,15 @@ export const buildFLUXGraph = async (
       originalSize,
       scaledSize,
       bbox,
-      1 - img2imgStrength,
+      denoisingStart,
       false
     );
+    if (optimizedDenoisingEnabled) {
+      g.updateNode(noise, {
+        denoising_start: 0,
+        trajectory_guidance_strength: img2imgStrength,
+      });
+    }
```

Comment:
We did this in a few places:
```suggestion
    if (optimizedDenoisingEnabled) {
      g.updateNode(noise, {
        denoising_start: 0,
        trajectory_guidance_strength: 1 - img2imgStrength,
      });
    }
```